package.json

```diff
@@ -26,49 +26,71 @@
     "test": "tests"
   },
   "contributors": [
-    "Jordan Morano <jordan@icarus.works>"
+    "Jordan Morano <jordan@icarus.works>",
+    "Bryan Crotaz <bgcrotaz@hotmail.com>"
   ],
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
+    "prepack": "ember ts:precompile",
+    "postpack": "ember ts:clean"
   },
   "dependencies": {
```

Comment:
Not sure, but do we need to explicitly mark `ember-modifier` as a dependency here since we are using it vs relying on any implicit inclusion from other dependencies?

Reply:
Yes. My mistake